libnavigation-metrics/api/current.txt

```diff
@@ -0,0 +1,22 @@
+// Signature format: 3.0
+package com.mapbox.navigation.metrics {
+
+  public final class MapboxMetricsReporter implements com.mapbox.navigation.base.metrics.MetricsReporter {
+    method public void addEvent(com.mapbox.navigation.base.metrics.MetricEvent metricEvent);
+    method public static void disable();
+    method public static void init(android.content.Context context, String accessToken, String userAgent);
+    method public void removeObserver();
+    method public void setMetricsObserver(com.mapbox.navigation.base.metrics.MetricsObserver metricsObserver);
+    method public static void toggleLogging(boolean isDebugLoggingEnabled);
+    field public static final com.mapbox.navigation.metrics.MapboxMetricsReporter! INSTANCE;
+  }
+
+}
+
+package com.mapbox.navigation.metrics.extensions {
+
+  public final class MetricEventExKt {
```

Comment:
Could we avoid tracking this empty `public` class? 🤔 There are only top-level functions